pypeit/par/pypeitpar.py

```diff
@@ -2042,6 +2043,13 @@ def __init__(self, use_flat=None, extrap_blu=None, extrap_red=None, samp_fact=No
                                   'entered as a list of integers, they should be converted to ' \
                                   'the detector name.  **Cannot be used with detector mosaics.**'
 
+        defaults['trim_std_pixs'] = None
+        dtypes['trim_std_pixs'] = [list, tuple]
+        descr['trim_std_pixs'] = 'List or tuple of two integers specifying the number of pixels to trim' \
```

Comment:
Should we include some checks in the `validate` method to make sure it's just a two element list? I appreciate this is done elsewhere in the code, but might be a good idea to do this first when the parameters are loaded.